setup.py

```diff
@@ -27,7 +27,7 @@
         "click>=8.0.3",  # used in scl
         "eclipse-sumo==1.10.0",  # sumo
         "gym==0.19.0",
-        "numpy>=1.19.5",
+        "numpy>=1.20.0",

```

Comment:
This needs to be set to `1.19.5` again. Seems like this PR includes the changes of PR #1319. Please merge develop into this branch to remove them.